### ISSUE TITLE
neural-networks subject incorrect environment setup

### ISSUE THREAD

**Issue by jarmo-seljamaa** [01-edu/public#2335]
neural-networks
---
[Exercise 3](https://github.com/01-edu/public/tree/master/subjects/ai/neural-networks#exercise-3-log-loss) includes the following suggestion:

> Check that `log_loss` from `sklearn.metrics` returns the same result

However, the virtual environment for this day does not include `scikit-learn` as a required library.
